### DISCUSSION TITLE
React-PDF version 1.6.8 not loading images

### DISCUSSION THREAD

**Discussion by NkosiShezi** [diegomura/react-pdf#2046] [Q&A]
Hi, I need help here whenever I try to pass a src to Image i get this error below . I'm not sure what it means 
<img width="465" alt="Screenshot 2022-10-09 at 08 56 15" src="https://user-images.githubusercontent.com/13049618/194742503-2197944c-199c-47af-9169-30c297f80203.png">

**Comment by CoskunCakir** [comment]
Hi @NkosiShezi, 

I see that you are using the svg extension which is not supported by the Image component. You can only use .jpg, .png, or base64 encoded image strings with the Image component.
for reference: https://react-pdf.org/components#image

There is this Svg component that you can use with the svg elements but I'm not sure you can use it with version 1.6.8 so you might need to upgrade your version.
for reference: https://react-pdf.org/svg

good luck,
cheers

**Reply by NkosiShezi** [reply]
Thanks for the response @coskuncakir Ill try to upgrade, its just strange as I get the same error even for .jpg and .png It just happens when i try to use an Image on my localhost  and staging site.. any image link i get on google works though. Im not sure if this old version does not support local imports...
<img width="576" alt="Screenshot 2022-10-10 at 07 59 07" src="https://user-images.githubusercontent.com/13049618/194806068-c110dbf1-f3ef-4174-8e2a-3bc7b5531eff.png">
<img width="576" alt="Screenshot 2022-10-10 at 07 47 30" src="https://user-images.githubusercontent.com/13049618/194806080-bdfe32ea-33b9-4abd-a69e-d6b0eaa338bf.png">

**Reply by CoskunCakir** [reply]
hey @NkosiShezi, that looks strange yeah. it shouldn't be a local import problem, have you checked that you are able to access your image from the URL you provided if yes I can't see any problem.

**Comment by diegomura** [comment]
This is an old version now
